courses/migrations/0036_add_partner_model.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.20 on 2023-08-28 11:44
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("courses", "0035_limit_certificate_revision_choices"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Partner",
```

Comment:
For consistency with MIT Open, can we call this "Platform" instead of "Partner"

Sorry I didn't bring this up earlier.